config/jobs/kubernetes/sig-cloud-provider/gcp/gpu/gpu-gce.yaml

```
@@ -18,7 +18,7 @@ periodics:
     preset-ci-gce-device-plugin-gpu: "true"
   annotations:
     fork-per-release: "true"
-    fork-per-release-generic-suffix: "true"
+    fork-per-release-replacements: "--extract=ci/latest -> --extract=ci/latest-{{.Version}}"
```

I think this job should be pointed at `ci/latest-fast` since it's on release-master-blocking?

---

ref: https://github.com/kubernetes/test-infra/pull/18290#discussion_r461858097

---

Opened https://github.com/kubernetes/test-infra/pull/18533 for this but forgot to update the replacements
```suggestion
    fork-per-release-replacements: "--extract=ci/latest-fast -> --extract=ci/latest-{{.Version}}"
```